NEWS.md

```diff
@@ -20,8 +20,8 @@ Released %TIMESTAMP%
 - Java API can now be used on Android
 
 ### Notes:
-- Removed outdated `-Gws` option of `opendds_idl`. This option produced sample
-  descriptions for the Wireshark dissector before OpenDDS v3.7.
+- Removed outdated `-Gws` option of `opendds_idl`. This option produced topic
```

Comment:
I think this is already changed in the other PR, may be easiest to leave it alone here to prevent conflicts.

Reply:
That's fine